### PR TITLE
Add solution

### PR DESCRIPTION
**Pull request by WaveOfDandelions**

### CODE REVIEW

**Review by choeqq — COMMENTED** [review]
Keep your PRs open so we can review them 😄 

@WaveOfDandelions

File: src/figuresClasses.ts [hunk 1]
```diff
@@ -1,19 +1,78 @@
...
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = Shape.Triangle;
```
Comment (asakevych) [inline src/figuresClasses.ts:11]: where do you have Shape.Triangle? I think you don't have such an object 😉 
you can write it the same as you have in other figures - `shape: Figure['shape'] = 'triangle';`

**Review by asakevych — COMMENTED** [review]
Good job! 🚀  just small change left 😉

File: src/figuresClasses.ts (hunk 1 above)
Comment (WaveOfDandelions) [reply src/figuresClasses.ts]: Fixed

**Review by choeqq — APPROVED** [review]
Looks good to me ✅

**Pull request closed without merging**